net/tac_plus/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=tac_plus
+PKG_VERSION:=4.0.4.28
+PKG_REV:=f28244cdc114ef88e62e2f754dc12b58bcfaa040
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_REV).tar.gz
```

Comment:
This is don't like. So the issue is that the file will be: tac_plus-4.0.4.28-hash. But the problem is that hashes are not alphabetical and this might break opkg update.

An alternative would be to specify PKG_SOURCE_DATE and PKG_SOURCE_VERSION and have PKG_SOURCE be: $(PKG_NAME)-$(PKG_SOURCE_DATE)-$(PKG_SOURCE_VERSION).

See https://github.com/openwrt/packages/blob/master/libs/quasselc/Makefile for an example.